app/controllers/candidate_interface/work_history/length_controller.rb

```diff
@@ -10,6 +10,8 @@ def submit
       @work_details_form = WorkHistoryForm.new(work_history_form_params)
 
       if @work_details_form.valid?
+        current_application.update!(feature_restructured_work_history: false)
```

Comment:
Prior to the flag being switched on, all candidates will be redirected to the existing flow. This allows us to ensure applicants that add jobs post between the feature_restructured_work_history column being deployed and us switching the flag use the old flow.